### ISSUE TITLE
Still able to be used?

### ISSUE THREAD

**Issue by YungGodChris** [jmquigs/ModelMod#26]
Saw this program was mainly for DX9 and was wondering if it still works for DX11 and if it is able to change gear assets etc? A little new to how this works with Guild Wars 2

**Comment by arb1911** [comment]
Yeah I still use it all the time it works well with DX9. There's no DX11 version yet, though.

**Comment by arsipsik** [comment]
I had the same question as I've been unable to get this to work with Guild Wars 2. @arb1911 would you be able to provide some information on how you get this working?
